Alexis neurosity

alexisbader opened the pull request:
## BCI-integration
[Integrate Neurosity and Blink detection]


<!-- Give a quick summary of the implementation details, provide design justifications if necessary -->
## Implementation description
Integrated the Unity-Neurosity sdk into our repo
Added a script for blink detection in the Blink detection folder in scripts 
Modified that RawBrainwaves Handler to call our blink detection and print BLINK DETECTED when the conditions are met


<!-- What should the reviewer do to verify your changes? Describe expected results and include screenshots when appropriate -->
## Steps to test
1. Make sure you have git LFS set up 
2. Install the dependencies needed in the README 
3. Open the game and make sure there are no errors in the console that prevent you from running


<!-- Draw attention to the substantial parts of your PR or anything you'd like a second opinion on -->
## What should reviewers focus on?
Will make a new ticket to integrate in the actual game once an integration scene is created


## Checklist
- [ ] My PR name is descriptive
- [ ] My commit messages are descriptive. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [ ] I have run the appropriate linter(s)
- [ ] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR
- [ ] My code is organized, and commented and documented so that another dev can understand what I did

kimchiloof commented:
I'm assuming the dependencies needed downloading are in [this](https://github.com/WATOLINK/bci-gaming/blob/5fd137ca87e8bfabe7ec102f6ebe822838ebcddc/bci-game/Assets/ExternalDependencyManager/Editor/README.md) readme?

Which ones are needed? The readme isn't too clear on which ones we need 😅 (also not too sure why I'm getting an error saying I need the iOS packages if we aren't using iOS specifically? idk)

Also wondering, can they not be added to our package manifest?

alexisbader commented:
> I'm assuming the dependencies needed downloading are in [this](https://github.com/WATOLINK/bci-gaming/blob/5fd137ca87e8bfabe7ec102f6ebe822838ebcddc/bci-game/Assets/ExternalDependencyManager/Editor/README.md) readme?
> 
> Which ones are needed? The readme isn't too clear on which ones we need 😅 (also not too sure why I'm getting an error saying I need the iOS packages if we aren't using iOS specifically? idk)
> 
> Also wondering, can they not be added to our package manifest?

Good point. I added them to the manifest and updated the pr. Im pretty sure the ios thing is just a warning error but lmk if you still see it now
